pkg/issuer/acme/setup.go

```diff
@@ -204,9 +204,8 @@ func (a *Acme) Setup(ctx context.Context) error {
 
 		// set the external account binding
 		eabAccount = &acmeapi.ExternalAccountBinding{
-			KID:          eabObj.KeyID,
-			Key:          eabKey,
-			KeyAlgorithm: string(eabObj.KeyAlgorithm),
+			KID: eabObj.KeyID,
+			Key: eabKey,
```

Comment:
I think this is okay to be honest - time will tell, and an early alpha would be a good idea (with some specific call out that if you are using an EAB based server, to go and give feedback)

Reply:
The issuer can still be applied with the `keyAlgorithm` field set, it will just be ignored. 

This I guess again is a question of whether we assume someone might be importhing this struct from `pkg/acme`.

But as you say, we won't be able to use upstream otherwise.

Reply:
I don't see external importers https://pkg.go.dev/github.com/jetstack/cert-manager/pkg/acme?tab=importedby, but as Ashley said earlier, someone of course might be importing it in closed source module.

But I don't see another way of doing this?